packages/@dcl/sdk/cli/commands/build/helpers.ts

```diff
@@ -0,0 +1,101 @@
+import { resolve } from 'path'
+
+// import spinner from '../../utils/spinner'
+import { exists, readdir, readFile } from '../../utils/fs'
+import { exec } from '../../utils/exec'
+import { Dict, hasPrimitiveKeys } from '../../utils/object'
+
+/**
+ * Required project dirs/files
+ */
+export const REQUIRED_FILES = {
+  PACKAGE_JSON: 'package.json',
+  TS_CONFIG_JSON: 'tsconfig.json',
+  SCENE_JSON: 'scene.json'
+}
+
+/**
+ * Required "package.json" structure
+ */
+export const REQUIRED_PACKAGE_JSON = {
+  devDependencies: {
+    '@dcl/sdk': ['latest']
```

Comment:
is `latest` mandatory ? Do we check this string or is just some context ? 
Because you can have a pinned version of the sdk

Reply:
we just have to defined what versions we are going to allow. It's just a placeholder